.travis.yml

```diff
@@ -5,11 +5,19 @@ arch:
 branches:
   only: master
 language: node_js
-before_install: npm install --global npm
+before_install: |
```

Comment:
@achrinza, since we're not relying on Travis CI anymore, perhaps we can remove this file. WDYT?

Reply:
We're no longer relying on Travis CI for x86, but we're leveraging it for other architectures (arm64, ppc64le, s390x). Since Node.js supports C++ add-ons, the portability of Node.js across architectures isn't as guaranteed. So I think we should keep Travis for the alternate archs.

I'm not sure if we need to test against all "supported" Node.js versions since we're already doing that with GH Actions Linux tests, or if we can slim it down to just Active LTS (v16).

WDYT?

Reply:
I see. Then let's keep it then.
Last time I checked, we're running out of the OSS credits on Travis. I can look into requesting for more credits.

Reply:
These alternate architectures don't incur credit for FOSS repos :-)

See: https://docs.travis-ci.com/user/billing-overview/#partner-queue-solution